### DISCUSSION TITLE
v1.5.1

### DISCUSSION THREAD

**Discussion by GeckoEidechse** [R2Northstar/Northstar#202] [Release]
**Bugfixes:**
Sticks and Stones: Fixed top player highlight when the leader exits the match. ([#197](https://github.com/R2Northstar/NorthstarMods/pull/197))
Fixed assist crashes ([#215](https://github.com/R2Northstar/NorthstarMods/pull/215))
Fixed battery bug in LTS ([#218](https://github.com/R2Northstar/NorthstarMods/pull/218))
Fixed tethers not despawning on round end ([#219](https://github.com/R2Northstar/NorthstarMods/pull/219))
Log players UUID on connect. ([#220](https://github.com/R2Northstar/NorthstarMods/pull/220))
Fix LTS crash related to viewentity
Fixed assists not updating
New spawnzone algorithm for CTF
Updated the localization files

**Contributors:**
@abarichello  @ASpoonPlaysGames @BobTheBob9 @Coopyy @cpdt @Dinorush

<hr /><em>This discussion was created from the release <a href='https://github.com/R2Northstar/Northstar/releases/tag/v1.5.1'>v1.5.1</a>.</em>

**Comment by ReaperKing36** [comment]
When I try to launch the client it says I am missing tier0.dll, can someone help?

**Reply by GeckoEidechse** [reply]
Most likely reason is you didn't install it properly. Check the [wiki troubleshooting page](https://r2northstar.gitbook.io/r2northstar-wiki/installing-northstar/troubleshooting#tier0) and [install instructions](https://r2northstar.gitbook.io/r2northstar-wiki/installing-northstar/basic-setup).

Alternatively use one of [the community provided installers](https://r2northstar.gitbook.io/r2northstar-wiki/installing-northstar/northstar-installers).

**Reply by FusionJ** [reply]
![image](https://user-images.githubusercontent.com/100502475/155867867-5f6f0db3-ad3f-4964-a0ed-bec24e166e4f.png)

You broke it and now it doesn't work

**Comment by Knull-Debug** [comment]
Hey, a question here 

I'm having trouble starting up Northstar due to an error (Included in image). I purchased the base Titanfall 2 game through an EA Play subscription if that matters. 
![image_2022-03-13_195208](https://user-images.githubusercontent.com/101533628/158087549-34c14ca7-83da-41b4-b13e-aa97d7f4d979.png)

**Reply by GeckoEidechse** [reply]
> I purchased the base Titanfall 2 game through an EA Play subscription if that matters.

That shouldn't really matter. Just make sure you've run the vanilla game at least once before using Northstar.

I'd suggest opening a ticket on the [Northstar Discord server](https://northstar.tf/discord). Add the screenshot and any other necessary info there and hopefully someone can help you out <3

**Comment by SnapBackToRealT** [comment]
Where is the *tier0.dll*.I cant find that! (Im Taiwanese，bad Eng sorry

**Comment by TheMuffinMan563** [comment]
Doesn't work, get the LSX Authentication Failed error -https://r2northstar.gitbook.io/r2northstar-wiki/installing-northstar/troubleshooting#lsx. It seems both the base game and the mod is broke. Shame, i used to play it fine last year.

**Comment by Depobutbad** [comment]
whenever I try to launch the actual multiplayer on the client it says that there are authentication problems and I dont know how to fix it

**Reply by emma-miler** [reply]
The servers are currently down due to a security issue with the base game. Check the discord announcements channel for any updates ^^

**Comment by geoffrygeorge** [comment]
Updated the mod version to 1.5.1. An error comes up saying authentication failed. Is it because the server is down currently?

**Reply by GeckoEidechse** [reply]
Yes. We will announce when it's back up in the `#announcements` channel on the [Northstar Discord](https://northstar.tf/discord).

**Reply by HenkoHallows** [reply]
Same here

**Comment by khaled-jehad** [comment]
there is a problem every time I click lunch northstar it says : disconnect : authentication failed  I tryed reloading mods and authentication agreement then lunching it again but its doesn't work what should I do

**Reply by geoffrygeorge** [reply]
As discussed above, wait for the announcements, in the #announcements channel in discord and join the server. You will get tons of info over there.

**Comment by Yessman-E** [comment]
I injected the northstar files into my game files on steam and I keep getting this when I try to open up the game.
![image](https://user-images.githubusercontent.com/102570650/160528919-8d5f7218-5159-422b-813b-09d71dbeae73.png)

**Reply by x3Karma** [reply]
why did you inject it

**Reply by Yessman-E** [reply]
By that I mean I copied the files to the game files, I don't know why I said inject.
